assets/README.md

```diff
@@ -1 +1 @@
-Icons: https://react-icons.github.io/react-icons/icons/fa/
+# 🦬️ Supervisord-monitor Frontend
```

Comment:
Could you create a wiki page and put a link there? 

I also think it's worth linking to those wikis in this README